rpc/src/helpers.rs

```diff
@@ -125,62 +134,272 @@ pub struct BlockHeaderMonitorInfo {
     pub protocol_data: String,
 }
 
+pub enum MonitorRpc {
+    MempoolOperations,
+    Head,
+}
+
+#[derive(Copy, Clone, Debug)]
+pub struct MempoolOperationsQuery {
+    pub applied: bool,
+    pub refused: bool,
+    pub branch_delayed: bool,
+    pub branch_refused: bool,
+}
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct MonitoredOperation {
+    pub signature: String,
+    pub branch: String,
+    pub contents: Value,
+
+    #[serde(skip_deserializing)]
+    pub protocol: Option<String>,
+    #[serde(skip_serializing)]
+    pub hash: String,
+}
+
 pub struct MonitorHeadStream {
```

Comment:
@adonagy 
Ado, we should to refactor this streaming a little bit - see description of this PR